ponyville/style.css

```diff
@@ -0,0 +1,15 @@
+/*
+Theme Name: Ponyville
+Theme URI: 
+Author: Automattic
+Author URI: https://automattic.com/
+Description: Ponyville is a lightly-opinionated base theme for block-based or classic themes using Global Styles
+Requires at least: ?
```

Comment:
I wanna say at least 5.7. I think we can have the templates be loaded using `do_blocks()` before FSE is released and then do a major version update that removes the `do_blocks()` and brings back pure html templates so we can support FSE after it launches on core, then we update the requirement to that version of WordPress

Reply:
I love this idea